internal/controlplane/handlers_authz.go

```diff
@@ -241,31 +239,22 @@ func (s *Server) ListRoleAssignments(
 		return nil, status.Errorf(codes.Internal, "error getting role assignments: %v", err)
 	}
 
-	if flags.Bool(ctx, s.featureFlags, flags.IDPResolver) {
-		for i := range as {
-			identity, err := s.idClient.Resolve(ctx, as[i].Subject)
-			if err != nil {
-				// if we can't resolve the subject, report the raw ID value
-				zerolog.Ctx(ctx).Error().Err(err).Msg("error resolving identity")
-				continue
-			}
-			as[i].Subject = identity.Human()
+	// Resolve the display names for the subjects
+	mapIdToDisplay := make(map[string]string, len(as))
+	for i := range as {
+		identity, err := s.idClient.Resolve(ctx, as[i].Subject)
+		if err != nil {
+			// if we can't resolve the subject, report the raw ID value
```

Comment:
this comment is no longer valid is it? If we can't resolve the subject, we would then just use the zero value (empty string). Or did you mean to first initialize the map with the raw ID value before trying to resolve it?

Reply:
Addressed in https://github.com/stacklok/minder/pull/3710